Market Tools/dmac_strategy.py

```diff
@@ -121,8 +121,8 @@ def backtest_dmac(signals_df):
     # Overlay the plots
     portfolio_entry_exit_plot = total_portfolio_value * entry * exit
     portfolio_entry_exit_plot.opts(
-        title="Apple Algorithm - Total Portfolio Value",
+        title="Total Portfolio Value",
```

Comment:
[Low priority, not for this PR] : Properly annotated charts are important and you I can see that you strive to ensure this but were faced with a dilemma of not having the ticker symbol.  In a future enhancement the ticker symbol can be passed into run_analysis making it also available to display_strategies.  Alternatively, the ticker can be added to the pricing_data DataFrame on its initial retrieval.  Or better yet, the name of the company can be looked up and added to the DataFrame or passed as a parameter.  Any of these options would enable you to dynamically set the title.